src/commands/status.js

```diff
@@ -0,0 +1,118 @@
+const Listr = require('listr');
+
+const os = require('os');
+const publicip = require('public-ip');
+const prettyms = require('pretty-ms');
+const prettybyte = require('pretty-bytes');
+
+const BaseCommand = require('../oclif/command/BaseCommand');
+
+const UpdateRendererWithOutput = require('../oclif/renderer/UpdateRendererWithOutput');
+
+const MutedError = require('../oclif/errors/MutedError');
+
+const PRESETS = require('../presets');
+
+class StatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    {
+      preset,
+    },
+    flags,
+    dockerCompose,
+  ) {
+    const status = {
+      host: {},
+      docker: {},
+      masternode: {},
+    };
+    const tasks = new Listr([
+      {
+        title: 'Show host status',
+        task: () => (
+          new Listr([
+            {
+              title: 'Fetch hostname',
```

Comment:
Not sure that they should be a part of Listr. Maybe just use a table to output this data?

Reply:
You mean something like this? https://www.npmjs.com/package/table-layout

Reply:
This one looks better https://www.npmjs.com/package/table